src/neo/Ledger/Blockchain.cs

```diff
@@ -27,6 +26,7 @@ public class Import { public IEnumerable<Block> Blocks; public bool Verify = tru
         public class ImportCompleted { }
         public class FillMemoryPool { public IEnumerable<Transaction> Transactions; }
         public class FillCompleted { }
+        public class RelayResult { public IInventory Inventory; public VerifyResult Result; }
```

Comment:
```suggestion
        public class RelayResult : LocalNode.RelayDirectly { public VerifyResult Result; }
```

Reply:
No, no, no. Such inheritance is illogical.